scenarios/segmentation/01_training_introduction.ipynb

```diff
@@ -0,0 +1,969 @@
+{
```

Comment:
<p>[minor] maybe add a short sentence on 'fit_one_cycle', including the link to its documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/computervision-recipes/pull/544/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Added link to fastai